packages/nodes-base/credentials/ShopifyAccessTokenApi.credentials.ts

```diff
@@ -5,6 +5,8 @@ import type {
 	INodeProperties,
 } from 'n8n-workflow';
 
+import { apiVersion } from '../nodes/Shopify/Const';
```

Comment:
why this is defined in the node? I think we should define the apiVersion in the credentials and have the node reference it

Reply:
@ShireenMissi I did have it in the credentials originally but OAuth also needs to know about it and by doing this approach if we decide in the future to update the API Version we only need to make 1 change instead of 3 of 4.

Reply:
nodes already depend on credentials, so best keeping it that way and not introduce a new dependency where credentials need to know about nodes

Reply:
In that case I will go back to my initial plan of having it in the credential

Reply:
@ShireenMissi updated and tested 👍🏻